draft-yasskin-httpbis-origin-signed-exchanges-impl.md

```diff
@@ -890,20 +907,29 @@ and header fields, and a response payload.
 
 This content type consists of the concatenation of the following items:
 
-1. The ASCII characters "sxg1-b1" followed by a 0 byte, to serve as a file
+1. The ASCII characters "sxg1-b2" followed by a 0 byte, to serve as a file
    signature. This is redundant with the MIME type, and recipients that receive
    both MUST check that they match and stop parsing if they don't.
 
    Note: As this is a snapshot of a draft of
    {{?I-D.yasskin-http-origin-signed-responses}}, it uses a distinct file
    signature.
-1. 3 bytes storing a big-endian integer `sigLength`. If this is larger
-   than 16kB, parsing MUST fail.
+1. 2 bytes storing a big-endian integer `fallbackUrlLength`.
+1. `fallbackUrlLength` bytes holding a `fallbackUrl`, which MUST be an absolute
+   URL with a scheme of "https".
+
+   Note: The byte location of the fallback URL is intended to remain invariant
```

Comment:
This means the magic "sxg1-b2" must always remain 7 chars. Out of curiosity:
 - What's after `b9`? `bA`..`bZ`? Or do we move to `g1` (for gamma)? :P
 - What's v1 look like? "sxg1\0\0\0"?

Reply:
> - What's after b9? bA..bZ? Or do we move to g1 (for gamma)? :P

We hope to get to a standard before reaching bA :airplane: 

> - What's v1 look like? "sxg1\0\0\0"?

That is the idea.